src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -485,7 +396,7 @@ function EmojiPickerMenu(props) {
                 <TextInput
                     label={translate('common.search')}
                     accessibilityLabel={translate('common.search')}
-                    role={CONST.ROLE.PRESENTATION}
+                    role={CONST.ACCESSIBILITY_ROLE.TEXT}
```

Comment:
This looks like a bad merge – should be `CONST.ACCESSIBILITY_ROLE` is deprecated and `CONST.ROLE` should be used instead